src/main/java/com/ning/http/util/SslUtils.java

```diff
@@ -15,101 +15,43 @@
  */
 package com.ning.http.util;
```

Comment:
I was unable to find any usages of this class inside the jenkinsci org. Fine with me